test/CMakeLists.txt

```diff
@@ -61,7 +61,7 @@ if(MIOPEN_TEST_HALF)
         set(SKIP_TESTS test_gru test_rnn_vanilla test_lstm test_conv_igemm_dynamic)
     endif()
     if(MIOPEN_TEST_GFX908)
-       set(SKIP_TESTS test_immed_conv3d test_conv3d test_fusion_aux test_activation test_lrn_test test_ctc test_conv2d_bias test_conv3d_bias test_cba_inference test_cbna_inference test_pooling2d test_na_train test_na_inference test_bn_aux test_conv_igemm_dynamic)
+       set(SKIP_TESTS test_immed_conv3d test_conv3d test_fusion_aux test_activation test_lrn_test test_ctc test_conv2d_bias test_conv3d_bias test_cba_inference test_cbna_inference test_pooling2d test_na_train test_na_inference test_bn_aux test_conv_igemm_dynamic test_conv_for_implicit_gemm_mlir)
```

Comment:
MLIR test should be disabled for non-FP32 data types, in order to save CI resources.

Reply:
I'd need your help in ctest related logic. I didn't quite follow what's the right way (for SKIP_TESTS) to have it tested. I feel like I can disable everything because there's already a dedicated stage.

Reply:
Fixed.